rm_vk/rm_vk/Core/NetworkService/NetworkService.swift

```diff
@@ -30,4 +30,8 @@ final class NetworkService: NetworkServiceProtocol {
             complition: complition
         )
     }
+
+    func fetchNews(complition: @escaping (Result<ResponseNews, Error>) -> Void) {
+        networkCoreService.downloadJson(url: RequestType.news.urlString, complition: complition)
```

Comment:
```suggestion
        networkCoreService.downloadJson(url: RequestType.news.urlString, completion: completion)
```
ты серьезно?)
поправь везде пока никто не видит)

Reply:
мне подбросили

Reply:
я думал это не считается за ошибку) я тоже обратил внимание, но подумал что грамматические ошибки мы не разбираем